src/components/Admin/ListUser/constants.js

```diff
@@ -6,6 +6,7 @@ const COULMNS = [
     field: 'serialNum',
     sorter: false,
     cellStyle: {
+      width: '10%',
```

Comment:
This is off no use. Remove it. Please check if your code solves the problem, when you make changes. If you cannot fix this issue, open another describing the problem, so it can be fixed buy others and this PR can be merged.